### PR TITLE
chore: Minor documentation refactoring on the new changelog process.

### PR DESCRIPTION
**Pull request by marcosuma**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): https://jira.mongodb.org/browse/CLOUDP-243940

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contributing guides](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/README.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [ ] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [ ] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: contributing/changelog-process.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Changelog Process
 
+- [PR Changelog check](#pr-changelog-check)
 - [Changelog format](#changelog-format)
 - [Changelog entry guidelines](#changelog-entry-guidelines)
 - [Script for creating changelog entry files](#script-for-creating-changelog-entry-files)
-- [PR Changelog check](#pr-changelog-check)
 - [Unreleased section of CHANGELOG.md automatic update](#unreleased-section-of-CHANGELOG.md-automatic-update)
```
Comment (AgustinBettati) [inline contributing/changelog-process.md]: I believe the section you want to move to the top is `Script for creating changelog entry files` right?

Comment (marcosuma) [reply contributing/changelog-process.md]: I am sorry :/ yes let me fix that

**Comment by marcosuma** [comment]
Ref: https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/8720632791/job/23922580319?pr=2162 

Now error is printed like this:

```
2024/04/17 10:53:21 Changelog file .changelog/2162.txt not found due to the following reason: open .changelog/2162.txt: no such file or directory.
Have you ran the `make generate-changelog-entry` command?
If this PR doesn't need a changelog entry file, consider using label skip-changelog-check.
Read contributing guides (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md) for more info.
exit status 1
```

**Comment by lantoli** [comment]
I personally prefer to have the error detail (errFile) at the end of the message. It's normally additional info that you may not need, and I think we can get a clearer message.

File: tools/check-changelog-entry-file/main.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func main() {
 		return
 	}
 
-	log.Fatalf("Consider using label %s if this PR doesn't need a changelog entry file. Read contributing guides for more info.\nChangelog file not found: %s, err: %v", skipLabelName, filePath, errFile)
+	log.Fatalf("Changelog file %s not found due to the following reason: %v.\nHave you ran the `make generate-changelog-entry` command?\nIf this PR doesn't need a changelog entry file, consider using label %s.\nRead contributing guides (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md) for more info.", filePath, errFile, skipLabelName)
```
Comment (marcosuma) [inline tools/check-changelog-entry-file/main.go]: @lantoli do you prefer this more? Feel free to suggest a proposal as well: end goal is to make sure the author quickly sees the command to run once they run into a failure of this check.

```suggestion
	log.Fatalf("Have you ran the `make generate-changelog-entry` command?\nIf this PR doesn't need a changelog entry file, consider using label %s.\nRead contributing guides (https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md) for more info.\nChangelog entry file %s not found due to the following reason: %v.", skipLabelName, filePath, errFile)
```

Comment (lantoli) [reply tools/check-changelog-entry-file/main.go]: yeah, that's good for me, knitpit:   Changelog file -> Changelog entry file. (agustin told me this in multiple comments with good reason :-) )

**Review by lantoli — APPROVED** [review]

**Review by oarbusi — APPROVED** [review]

**Review by lantoli — APPROVED** [review]

**Review by AgustinBettati — APPROVED** [review]
LGTM

**Pull request merged**